CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+## Before you start
+* Read our [FAQ](https://github.com/OML-Team/open-metric-learning#faq).
+* Check out [python examples](https://github.com/OML-Team/open-metric-learning#get-started-using-python)
+  and [config examples](https://github.com/OML-Team/open-metric-learning/tree/main/examples).
+
+## Choosing a task
+* You can work on one of the [existing issues](https://github.com/OML-Team/open-metric-learning)
+  or create the new one. Especially pay attention to the issues marked with the `good_first_issue` flag.
+* Start the conversation under the issue that you picked. We will discuss the design and content of the pull request and
+  then you can start working on it.
+
+## Contributing
+* Fork the repository.
+* Clone it locally.
+* Create a branch with a name that speaks for itself.
+* Set up the environment. You can install the library in dev mode via `pip install -e .`
+  or [build / pull docker image](https://github.com/OML-Team/open-metric-learning#installation).
```

Comment:
We need to actualize docker before suggesting developing in this way

Reply:
discussed offline